test/libsolidity/semanticTests/multiSource/multiple_imports_multiple_alais.sol

```diff
@@ -0,0 +1,21 @@
+==== Source: s1.sol ====
+function f() pure returns (uint) { return 1337; }
+function g() pure returns (uint) { return 42; }
+==== Source: s2.sol ====
+import {f as g} from "s1.sol";
+==== Source: s3.sol ====
+// imports f()->1337 as g()
+import "s2.sol";
+// imports f()->1337 as f() and
+// g()->42 as g
+import {f as f, g as g} from "s1.sol";
+contract C {
+  function foo() public pure returns (uint) {
+    // calls f()->1337 / f()->1337
+    return f() / g();
+  }
+}
+// ====
+// compileViaYul: also
+// ----
+// foo() -> 1
```

Comment:
Should this test pass? Question is: What is the precedence for symbol aliases that may be redeclared? Current behavior is first definition is "final" and susbsequent redeclarations do not have an effect